.gitignore

```diff
@@ -41,4 +41,4 @@ yarn-error.log*
 .idea
 
 # main file generated by database script
-.main
\ No newline at end of file
+.main
```

Comment:
is it .main? Should we use dot here?

I already add it by the way. You can remove this if you want.